README.md

```diff
@@ -138,66 +138,62 @@ paths:
             '*/*':
               schema:
                 type: string
+                example: hello world
 servers:
-  - url: '{schema}://{host}/{basePath}'
-    variables:
-      schema:
-        default: http
-        enum:
-          - http
-          - https
-      host:
-        default: stoplight.io
-        enum:
-          - stoplight.io
-          - dev.stoplight.io
-      basePath:
-        default: api
+  - url: https://stoplight.io/api
+    name: Production
+  - url: https://stag.stoplight.io/api
+    name: Staging
 ```
 
 You can make a request enforcing server validation by providing the `__server` query string parameter:
 
 ```bash
-curl -X GET localhost:4010/pet?__server=http://stoplight.io/api
-# This will return a 200 response
+curl -X GET localhost:4010/pet?__server=https://stoplight.io/api
+hello world
 ```
 
 On the other hand, putting a server which is not defined in the specification, for example:
 
 ```bash
-curl -X GET localhost:4010/pet?__server=ftp://acme.com/api
+curl -X GET localhost:4010/pet?__server=https://nonsense.com/api
 ```
 
-Will give you the following error
+Will give you the following error:
 
 ```json
 {
   "type": "https://stoplight.io/prism/errors#NO_SERVER_MATCHED_ERROR",
   "title": "Route not resolved, no server matched.",
   "status": 404,
-  "detail": "The base url ftp://acme.com/api hasn't been matched with any of the provided servers"
+  "detail": "The server url http://nonsense.com/api hasn't been matched with any of the provided servers"
 }
 ```
 
 ## What's next for Prism?
 
 - [x] Server Validation
-- [x] Accept header validation
-- [ ] Content header validation
+- [x] Content Negotiation
 - [ ] Security Validation
-- [ ] Dynamic Mocking (use JS to script custom interactions)
-- [ ] Forwarding proxy with validation
-- [ ] Recording traffic to spec file
-- [ ] Data Persistence (turn Prism into a sandbox server)
-- [x] Support files ending with `.yml` and extension-less files
+- [ ] Custom Mocking
+- [ ] Validation Proxy
+- [ ] Recording / "Learning" mode to create spec files
+- [ ] Data Persistence (allow Prism act like a sandbox)
 
 ## FAQs
 
-**Requests with base paths are failing. Why?**
+**Why am I getting 404 errors when I include my basePath?**
 
-Base paths are completely ignored by the Prism HTTP server, so they can be removed from the request.
-If you have a base path of `/api` and your path is defined as `hello`, then a request to
-`http://localhost:4010/hello` would work, but `http://localhost:4010/api/hello` will fail.
+OpenAPI v2.0 had a concept called "basePath", which was essentially part of the HTTP path the stuff
```

Comment:
Same here with the newlines. I think we can run prettier on markdown files as well, maybe that'll help